snippets/csharp/VS_Snippets_CLR_System/system.string.isnullorempty/cs/isnullorempty1.cs

```diff
@@ -4,18 +4,18 @@ public class Example
 {
     public static void Main()
     {
+        // <Snippet1>
```

Comment:
I don't quite follow this change. I think you've done what I requested before no?